projects/ng2-date-picker/.eslintrc.json

```diff
@@ -10,8 +10,8 @@
       ],
       "parserOptions": {
         "project": [
-          "projects/ng2-datepicker/tsconfig.lib.json",
-          "projects/ng2-datepicker/tsconfig.spec.json"
+          "projects/ng2-date-picker/tsconfig.lib.json",
```

Comment:
how is that relevant to your changes? I think that you pulled from older version and not from the one in the master

Reply:
Imports are wrong, so the editor gives errors. You can see these wrong imports.

https://github.com/vlio20/angular-datepicker/blob/master/projects/ng2-date-picker/.eslintrc.json